src/api/bodhi_utils.js

```diff
@@ -0,0 +1,13 @@
+const { Utils } = require('qweb3');
+
+const BodhiUtils = {
+  async isValidQTUMAddress(args) {
+    const {
+      address,
+    } = args;
+
+    return new Utils(isQtumAddress(address));
```

Comment:
lets use `qweb3.getHexAddress()`. it has better validation on the address using qtumd. if you are unable to convert it, then the qtum address is invalid.